lib/src/material_color_picker.dart

```diff
@@ -77,47 +77,48 @@ class _MaterialColorPickerState extends State<MaterialColorPicker> {
     });
   }
 
-  ColorSwatch _findMainColor(Color shadeColor) {
-    for (final mainColor in _colors)
+  ColorSwatch? _findMainColor(Color shadeColor) {
+    for (final ColorSwatch? mainColor in _colors!)
```

Comment:
please remove the explicit type for `mainColor`. It is inferred as not nullable, so explicitly adding a nullable type is counterproductive.